### ISSUE TITLE
WCAG: Empty alt attributes should be allowed on img tags

### ISSUE THREAD

**Issue by niektenhoopen** [nystudio107/craft3-imageoptimize#411]
### Describe the bug
Screen readers announce the filename of decorative images if the alt tag is not present. The plug-in should allow empty alt tags.

https://www.w3.org/WAI/tutorials/images/decorative/#:~:text=Leaving%20out%20the%20alt%20attribute,the%20image%20on%20the%20page.

### Expected behaviour
There should be an `alt=""` on images if it's specified as empty

### Versions
- Plugin version:
- Craft version:

**Comment by khalwat** [comment]
Thanks, I'll have a look!

**Comment by khalwat** [comment]
Implemented via your PR, which I modified a bit and backported to the Craft 3 version.

Craft CMS 3:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-imageoptimize": "dev-develop as 1.6.56”,
```
Then do a `composer clear-cache && composer update`

…..

Craft CMS 4:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-imageoptimize": "dev-develop-v4 as 4.0.10”,
```
Then do a `composer clear-cache && composer update`

…..

Craft CMS 5:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-imageoptimize": "dev-develop-v5 as 5.0.3”,
```
Then do a `composer clear-cache && composer update`
